.github/copilot-instructions.md

```diff
@@ -0,0 +1,262 @@
+
+### Style Guide for Japanese Documentation
+
+#### Writing Style (日本語ドキュメント)
+- **敬語**: 基本的に丁寧語を使用（です・ます調）
+- **技術用語**: 英語の技術用語はそのまま使用し、必要に応じて日本語で補足
+- **読者層**: Unityに慣れているが、Modular Avatarは初心者という前提
+- **文体**: 簡潔で分かりやすく、実用的な情報を重視
+
+#### 構造要件
+- 英語版と同じ見出し構造を維持し、同じ名前付きアンカーを使用
+- フロントマターも英語版と同じ構造を保持
+- 画像やリンクは英語版と同じものを参照
+
+#### 翻訳のガイドライン
+- 直訳ではなく、日本語として自然な表現を心がける
+- 文化的な違いを考慮した説明を適宜追加
+- VRChat日本語コミュニティでよく使われる用語を優先
+- 英語版の更新に追従し、内容の同期を保つ
+
+### Documentation Consistency Requirements
+
+#### Language Synchronization
+- Japanese documentation must mirror the English structure exactly
+- Both versions must use identical named anchors for cross-linking consistency
+- Update both language versions simultaneously when making documentation changes
+- Ensure images and diagrams are accessible from both language versions
+
+#### Cross-Language Linking
+- Use named anchors to ensure stable URLs across languages
+- Test links in both languages after updates
+- Maintain parallel navigation structures in both `sidebars.js` and Japanese i18n
+
+#### Image and Media Guidelines
+- Store images in the English docs directory and reference from both languages
+- Use descriptive filenames that indicate the content
+- Provide appropriate alt text for accessibility
+- Optimize images for web display while maintaining clarity
+
+## Testing and Quality Assurance
+
+### Unit Testing
+- Write unit tests for all core business logic in `UnitTests~/`
+- Inherit from `TestBase` class for common test setup and teardown
+- Use Unity Test Runner with NUnit framework
+- Mock Unity objects appropriately in tests using `CreatePrimitive()` or prefabs
+- Test both editor and runtime behavior where applicable
+- Use conditional compilation (`#if MA_VRCSDK3_AVATARS`) for VRChat-specific tests
+- Clean up created GameObjects in teardown methods to prevent test pollution
+- Use the established pattern for loading test assets via GUID references
+
+### Documentation Testing
+- Build documentation locally before committing: `cd docs~ && yarn build`
+- Test both English and Japanese versions
+- Verify all links work in both languages
+- Check that named anchors resolve correctly
+
+### Integration Testing
+- Test components in Unity with real avatar setups
+- Verify non-destructive behavior (original assets unchanged)
+- Test build processes with NDMF integration
+- Validate VRChat SDK compatibility
+
+## Build and Development Workflow
+
+### Documentation Development
+```bash
+# Navigate to docs directory
+cd docs~
+
+# Install dependencies
+yarn install
+
+# Start development server
+yarn start
+
+# Build for production
+yarn build
+
+# Type checking
+yarn typecheck
+```
+
+### Unity Development
+- Use Unity 2022.3.22f1 (as specified in project)
+- Test with VRChat SDK3 Avatars
+- Verify NDMF integration works correctly
+- Test both play mode and edit mode functionality
+
+## Common Patterns and Anti-Patterns
+
+### ✅ Good Practices
```

Comment:
These practices duplicate content higher up in the file - see comments over there.